src/scss/components/button/button.md

```diff
@@ -26,28 +26,33 @@
 </div>
 ```
 
-### Squared
-```html
-<button class="Btn Btn--squared">
-  <i class="Icon Icon--like"></i>
-</button>
 
-```
 
 ## Types
 
+### Raw
 ```html
 <button class="Btn Btn--raw">Raw button</button>
 ```
 
+### Rounded
+```html
+<button class="Btn Btn--rounded">
```

Comment:
You're right
rounded https://www.google.fr/search?q=button+circle&source=lnms&tbm=isch&sa=X&ei=3r-NVaTzOsGsU5GEi7gC&ved=0CAcQ_AUoAQ&biw=1276&bih=714#tbm=isch&q=button+rounded

vs 

circle https://www.google.fr/search?q=button+circle&source=lnms&tbm=isch&sa=X&ei=3r-NVaTzOsGsU5GEi7gC&ved=0CAcQ_AUoAQ&biw=1276&bih=714#tbm=isch&q=button+rounded